.github/workflows/go.yml

```diff
@@ -1,13 +1,40 @@
 name: Go
 on: [push]
 jobs:
+    build:
+        name: Build
+        runs-on: ubuntu-latest
+        steps:
 
-  build:
-    name: GoCity CI
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-go@v1
-        with:
-          version: '1.12'
-      - run: go test -race -cover ./...
+            - name: Set up Go 1.12
+              uses: actions/setup-go@v1
+              with:
+                  go-version: 1.12.4
```

Comment:
We have a new version available, 1.13.

Reply:
I have updated it to 1.13, in the latest commit.